frontend_react/src/components/Button.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Button({ btnName, btnColor = 'primary' }) {
+  return (
+    <div className={`bg-${btnColor} text-white p-2 rounded-lg font-medium hover:bg-white hover:text-primary duration-300 ease-in-out`}>{btnName}</div>
```

Comment:
Use button tag for Button. 
Try to use different HTML elements depending on their use case. Ex., for clicks, you should use a button; for links, you should use a tag; for grouping, you can use div, etc. It helps in [web accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility).